modular_skyrat/code/modules/mob/emote.dm

```diff
@@ -58,3 +58,13 @@
 	else
 		message = "makes a very loud noise."
 	. = ..()
+
+/datum/emote/flip/run_emote(mob/living/user, params) //no fun allowed :)
+	if(prob(20))
+		user.visible_message("<span class='warning'>[user] tries to flip, but lands flat on their face!</span>", "<span class='danger'>You try to flip, but land flat on your face!</span>")
+		user.Knockdown(20)
+		user.setDir(NORTH)
+		user.apply_damage(rand(1, 8), BRUTE, BODY_ZONE_HEAD)
+		user.adjustOrganLoss(ORGAN_SLOT_BRAIN, rand(1, 5), 20)
```

Comment:
```suggestion
```
Having brain damage here is definitely excessive.

Reply:
brain damage actually gets healed over time, and its capped at 20, which doesnt leave any permanent brain damage.